platform/app/services/temporal/activities/update_sync_status_activity.rb

```
@@ -0,0 +1,97 @@
+    class UpdateSyncStatusActivity < ::Temporal::Activity
+      retry_policy(
+        interval: 1,
+        backoff: 2,
+        max_attempts: 3
+      )
+
+      def execute(sync_run_id:, status:, error_message: nil)
+        @sync_run_id = sync_run_id
+        @status = status
+        @error_message = error_message
+        
```

rubocop (1.68.0)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)